tests/integration/helpers/helpers.go

```diff
@@ -0,0 +1,226 @@
+package helpers
+
```

Comment:
Not sure if all the functions here are still used

Reply:
Since this helpers file should be part of the vendor I'm not gonna remove the unused functions. Is this ok for you?